wiki/Help_Centre/en.md

```diff
@@ -737,9 +732,11 @@ If you have been inactive for a lengthy period of time and your statistics are s
 
 ### Installation
 
+*For more information, see: [Installation](https://osu.ppy.sh/help/wiki/Installation)*
```

Comment:
Most of the sections below looks like they might have been copied to the Installation article, I think it should be removed here since there is already a link to go to that article.

The registration section has a similar deal.